src/openrct2-ui/input/KeyboardShortcuts.h

```diff
@@ -24,101 +24,106 @@
-    SHORTCUT_QUICK_SAVE_GAME,
-    SHORTCUT_SHOW_OPTIONS,
-    SHORTCUT_MUTE_SOUND,
-    SHORTCUT_WINDOWED_MODE_TOGGLE,
-    SHORTCUT_SHOW_MULTIPLAYER,
-    SHORTCUT_PAINT_ORIGINAL_TOGGLE,
-    SHORTCUT_DEBUG_PAINT_TOGGLE,
-    SHORTCUT_SEE_THROUGH_PATHS_TOGGLE,
-    SHORTCUT_RIDE_CONSTRUCTION_TURN_LEFT,
-    SHORTCUT_RIDE_CONSTRUCTION_TURN_RIGHT,
-    SHORTCUT_RIDE_CONSTRUCTION_USE_TRACK_DEFAULT,
-    SHORTCUT_RIDE_CONSTRUCTION_SLOPE_DOWN,
-    SHORTCUT_RIDE_CONSTRUCTION_SLOPE_UP,
-    SHORTCUT_RIDE_CONSTRUCTION_CHAIN_LIFT_TOGGLE,
-    SHORTCUT_RIDE_CONSTRUCTION_BANK_LEFT,
-    SHORTCUT_RIDE_CONSTRUCTION_BANK_RIGHT,
-    SHORTCUT_RIDE_CONSTRUCTION_PREVIOUS_TRACK,
-    SHORTCUT_RIDE_CONSTRUCTION_NEXT_TRACK,
-    SHORTCUT_RIDE_CONSTRUCTION_BUILD_CURRENT,
-    SHORTCUT_RIDE_CONSTRUCTION_DEMOLISH_CURRENT,
-    SHORTCUT_LOAD_GAME,
-    SHORTCUT_CLEAR_SCENERY,
-    SHORTCUT_GRIDLINES_DISPLAY_TOGGLE,
-    SHORTCUT_VIEW_CLIPPING,
-    SHORTCUT_HIGHLIGHT_PATH_ISSUES_TOGGLE,
-    SHORTCUT_TILE_INSPECTOR,
-    SHORTCUT_ADVANCE_TO_NEXT_TICK,
-    SHORTCUT_SCENERY_PICKER,
-    SHORTCUT_SCALE_UP,
-    SHORTCUT_SCALE_DOWN,
-    SHORTCUT_INSERT_CORRUPT_ELEMENT,
-    SHORTCUT_COPY_ELEMENT,
-    SHORTCUT_PASTE_ELEMENT,
-    SHORTCUT_REMOVE_ELEMENT,
-    SHORTCUT_MOVE_ELEMENT_UP,
-    SHORTCUT_MOVE_ELEMENT_DOWN,
-    SHORTCUT_INCREASE_X_COORD,
-    SHORTCUT_DECREASE_X_COORD,
-    SHORTCUT_INCREASE_Y_COORD,
-    SHORTCUT_DECREASE_Y_COORD,
-    SHORTCUT_INCREASE_ELEM_HEIGHT,
-    SHORTCUT_DECREASE_ELEM_HEIGHT,
-    SHORTCUT_TOGGLE_CLEARANCE_CHECKS,
-
-    SHORTCUT_COUNT,
-
-    SHORTCUT_UNDEFINED = 0xFFFF,
+    ReduceGameSpeed,
+    IncreaseGameSpeed,
+    OpenCheatWindow,
+    RemoveTopBottomToolbarToggle,
+    ScrollMapUp,
+    ScrollMapLeft,
+    ScrollMapDown,
+    ScrollMapRight,
+    OpenChatWindow,
+    QuickSaveGame,
+    ShowOptions,
+    MuteSound,
+    WindowedModeToggle,
+    ShowMultiplayer,
+    PaintOriginalToggle,
+    DebugPaintToggle,
+    SeeThroughPathsToggle,
+    RideConstructionTurnLeft,
+    RideConstructionTurnRight,
+    RideConstructionUseTrackDefault,
+    RideConstructionSlopeDown,
+    RideConstructionSlopeUp,
+    RideConstructionChainLiftToggle,
+    RideConstructionBankLeft,
+    RideConstructionBankRight,
+    RideConstructionPreviousTrack,
+    RideConstructionNextTrack,
+    RideConstructionBuildCurrent,
+    RideConstructionDemolishCurrent,
+    LoadGame,
+    ClearScenery,
+    GridlinesDisplayToggle,
+    ViewClipping,
+    HighlightPathIssuesToggle,
+    TileInspector,
+    AdvanceToNextTick,
+    SceneryPicker,
+    ScaleUp,
+    ScaleDown,
+    InsertCorruptElement,
+    CopyElement,
+    PasteElement,
+    RemoveElement,
+    MoveElementUp,
+    MoveElementDown,
+    IncreaseXCoord,
+    DecreaseXCoord,
+    IncreaseYCoord,
+    DecreaseYCoord,
+    IncreaseElementHeight,
+    DecreaseElementHeight,
+    ToggleClearanceChecks,
+
+    Count,
+
+    Undefined = 0xFFFF,
 };
 
+constexpr size_t get_shortcuts_count()
```

Comment:
We Use 'PascalCase' for functions and 'camelCase' for variables.

Reply:
So the function with the underscore (_) are old ones?